src/forcegraph-kapsule.js

```diff
@@ -550,16 +554,16 @@ export default Kapsule({
 
           let lineMaterial = customLinkMaterialAccessor(link);
           if (!lineMaterial) {
-            if (!lineMaterials.hasOwnProperty(color)) {
-              const lineOpacity = state.linkOpacity * colorAlpha(color);
-              lineMaterials[color] = new three.MeshLambertMaterial({
+            if (!lineMaterials.hasOwnProperty(color+opacity)) {
```

Comment:
It's better to pre-compute this index key, to prevent repeating the calculation. Also, because `color` is not guaranteed to always be a string, the combo key may lead to unexpected results. Perhaps better to use literals instead of concat: 
``` `${color}${opacity}` ```.